NotIntegrationTests/ConditionalFact.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Xunit;
+
+namespace NotIntegrationTests
+{
+    public class ConditionalFact : FactAttribute
```

Comment:
A more specific name would let people know this is specifically about integration tests.